lib/elixir/lib/kernel/typespec.ex

```diff
@@ -888,6 +894,10 @@ defmodule Kernel.Typespec do
     typespec((quote do: :elixir.as_boolean(unquote(arg))), vars, caller)
   end
 
+  defp typespec({:keyword, _meta, args}, vars, caller) do
```

Comment:
I messed the fact the we have `[]` here as well, I think better be strict anyway with `length(args) =< 1`.